src/templates/docs.tsx

```diff
@@ -158,11 +218,9 @@ const components = {
   RollupProductDevDoc,
 }
 
-const Contributors = styled(FileContributors)`
-  @media (max-width: ${(props) => props.theme.breakpoints.l}) {
-    padding-bottom: 2rem;
-  }
-`
+const Contributors = (
+  props: ComponentPropsWithoutRef<typeof FileContributors>
+) => <FileContributors pb={{ base: 8, lg: 2 }} {...props} />
```

Comment:
We'll need to override the `p` with the same breakpoints we are using in the `pb` style, otherwise the `pb` is not going to be applied correctly.

```suggestion
) => <FileContributors p={{ base: 0, lg: 2 }} pb={{ base: 8, lg: 2 }} {...props} />
```